Common/config/log4js/logconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+	"appenders": {
+		"default": {
+			"type": "file",
+			"filename": "../onlyoffice.log",
```

Comment:
just confirming if this path is correct, its suppose to be in /opt/zimbra/log/ right? is it relative to that